src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -1063,11 +1066,26 @@ def test_storage_account_file_delete_retention_policy(self, resource_group, stor
             'cmd': 'storage account file-service-properties'
         })
         self.cmd('{cmd} show --account-name {sa} -g {rg}').assert_with_checks(
-            JMESPathCheck('shareDeleteRetentionPolicy', None))
+            JMESPathCheck('shareDeleteRetentionPolicy.enabled', True),
+            JMESPathCheck('shareDeleteRetentionPolicy.days', 7))
```

Comment:
Can we add this breaking change into history notes so that customers can be aware?

Reply:
sure